c2cgeoportal/scaffolds/create/print/print-apps/+package+/config.yaml.mako

```diff
@@ -15,18 +15,18 @@ templates:
                 size: 40
                 default:
                     graphic: "file:///north.svg"
-            scalebar: !scalebar &scalebar
-                width: 150
-                height: 20
-                default:
-                     fontSize: 8
             map: !map &map
                 maxDpi: 254
                 dpiSuggestions: [254]
                 zoomLevels: !zoomLevels
                     scales: [100, 250, 500, 2500, 5000, 10000, 25000, 50000, 100000, 500000]
                 width: 555
                 height: 675
+            scalebar: !scalebar &scalebar
+                width: 150
+                height: 20
+                default:
+                     fontSize: 8
```

Comment:
this change is not needed and I think that northArrow and scalebar should be behind :-)

Reply:
Regarding scalebar: we changed the order here too, so that it is consistent with the order for createMap / createScalebar. I think it is easier to read the file that way.

Regarding northArrow: are you sure? We applied the same changes as for Vaud, https://github.com/camptocamp/vaud_geomapfish/commit/119438438a282f88b084ad608f22df6408873561 
It works there (as far as we know).
Should we change the order, and if so, to which order exactly?

Reply:
I understand then to be clean for me it's better to also move the `northArrow`...